db/seeds.rb

```diff
@@ -466,8 +466,18 @@ def generate_objs(method, obj, n)
 # ============================================================================
 
 unless EquipmentItem.count == 0
-  n = MINIMAL ? 10 : ask_for_records('Reservation')
+  # TODO: change 100 back to 10
+  n = MINIMAL ? 100 : ask_for_records('Reservation')
   generate_objs(:generate_reservation, 'reservation', n)
+  # TODO: remove
```

Comment:
I left the below code for review; it's how I tested that my fix was working. I ran the seed script 20 times (generating 2,000 reservations altogether) and didn't get any errors. On `master` modified with the same testing code, I got 66 date errors in 100 runs.